ogr/services/github.py

```diff
@@ -66,8 +99,27 @@ def get_branches(self) -> List[str]:
     def get_description(self) -> str:
         return self.github_repo.description
 
-    def get_fork(self) -> Optional[GitProject]:
-        raise NotImplementedError
+    def get_fork(self, create: bool = True) -> Optional["GithubProject"]:
+        """
+        Provide GithubProject instance of a fork of this project.
+
+        Returns None if this is a fork.
+
+        :param create: create a fork if it doesn't exist
+        :return: instance of GithubProject
+        """
+        if self.is_fork:
+            return None
+        f = self.is_forked()
+        if not f:
```

Comment:
That's the reason I do not like one-letter variables and `is_something` returning an object...

Reply:
that's why we're using typing, right? You can always check what you're getting

Reply:
`Readability counts.` vs `Simple is better than complex.`..;-)

I just like the code that I can understand/review without any further observation or running the code.

Reply:
I renamed